pocs/utils/config.py

```diff
@@ -8,7 +8,46 @@
 
 
 def load_config(config_files=None, simulator=None, parse=True, ignore_local=False):
-    """ Load configuation information """
+    """Load configuation information
+
+    This function supports loading of a number of different files. If no options
+    are passed to `config_files` then the default `$POCS/conf_files/pocs.yaml`
+    will be loaded. See Notes for additional information.
+
+    Notes:
+        `config_files` supports a number of options:
+        * `config_files` is a list and loaded in order, so the first entry
+            will have any values overwritten by similarly named keys in
+            the second entry.
+        * Entries can be placed in the `$POCS/conf_files` folder and
+            should be passed as just the file name, e.g.
+            [`weather.yaml`, `email.yaml`] for loading
+            `$POCS/conf_files/weather.yaml` and `$POCS/conf_files/email.yaml`
+        * The `.yaml` extension will be added if not present, so list can
+            be writen as just ['weather', 'email'].
+        * `config_files` can also be specified by an absolute path, which
+            can exist anywhere on the filesystem.
+        * Local versions of files can override built-in versions and are
+            automatically loaded if placed in the `$POCS/conf_files` folder.
+            The files have a `<>_local.yaml` name, where `<>` is the built-in
```

Comment:
Seems like we should generate the local file name from the `path` variable at line 77, and not assume that the file was in config_dir. Specifically, replace in `path`  the trailing ".yaml" with "_local.yaml".

Reply:
Yes, I like the idea of saving to a local file more but would rather do it in a separate PR. 

> not assume that the file was in config_dir

Confused about this comment.

Reply:
Create https://github.com/panoptes/POCS/issues/238